rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/CoordinatorBookKeeper.java

```diff
@@ -61,6 +61,12 @@ public CoordinatorBookKeeper(Configuration conf, MetricRegistry metrics, Ticker
     registerMetrics();
   }
 
+  @Override
```

Comment:
Put this is `BookKeeper.java` as both the implementation in Coordinator and Worker are same